src/report.rs

```diff
@@ -256,12 +259,13 @@ pub fn remove() -> Result<()> {
 mod test {
     use super::*;
     use crate::directory::test::test_tree;
-    use crate::directory::Uuid;
 
     #[test]
     pub fn pdf() -> Result<()> {
+        simplelog::SimpleLogger::init(log::LevelFilter::Warn, simplelog::Config::default())
+            .unwrap();
         let tree = test_tree();
-        let roots = vec![*tree.root(Uuid::from(""))];
+        let roots = vec![*tree.root()];
```

Comment:
`root(&self)` previously took a `Uuid` as arg, the idea was that there would be multiple roots (*trash* for example is a root). This is not used and only the root `""` (empty string) is used. Therefore this change.